libs/common/src/middleware/set-no-store-cache-control.test.ts

```diff
@@ -0,0 +1,31 @@
+import request from 'supertest';
+import express, { Request, Response, NextFunction } from 'express';
+import { HttpStatusCode } from 'axios';
+import { setNoStoreCacheControl } from './set-no-store-cache-control';
+
+const app = express();
+app.use(setNoStoreCacheControl);
+
+afterEach(() => {
+  jest.resetAllMocks();
+});
+
+describe('set-no-store-cache-control', () => {
+  it('should set Cache-Control headers correctly', async () => {
+    const middleware = (req: Request, res: Response, next: NextFunction) => {
+      setNoStoreCacheControl(req, res, next);
+    };
+
+    const next = jest.fn((_req: Request, res: Response) => res.send());
+
+    const testApp = express();
+    testApp.use(middleware);
+    testApp.get('/', next);
```

Comment:
We want to test that the middleware itself is calling next, not the route controller. Would this work?
```suggestion
    testApp.get('/', middleware);
```